src/App.tsx

```diff
@@ -15,36 +16,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  alph = 'alph',
+  length = 'length',
+}
+
+interface SortParams {
+  sortField: string | number,
+  reversed: boolean
+}
+
+const getPreparedGoods = (goods: string[], { sortField, reversed }: SortParams) => {
+  let preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortType.alph:
+          return good1.localeCompare(good2);
+        case SortType.length:
+          return good1.length - good2.length;
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reversed) {
+    preparedGoods = preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+};
+
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState('');
```

Comment:
The `sortField` state is initialized as an empty string, which might not align with the `SortType` enum values. Consider initializing it with a valid `SortType` value or adjusting the logic to handle an empty string appropriately.